RELEASING.md

```diff
@@ -91,27 +91,27 @@ To push a release version
 
 7. To ensure a clean build, remove *all* Maven targets (including subdirectories not handled by `mvn clean`) by running `rm -rf target */target`.
 
-8. Run `utilities/stage_release.sh`.
-This script builds and stages the release artifact on the Maven Central Repository, updates the README.md files with the release version + commits them locally, and finally generates a new site version for the gh-pages branch under a temporary directory named `tmp_gh-pages`. If you haven't run the release process before, it's worth verifying everything; check the staged release on the Sonatype website, and verify that the local commits have the right version updates.
+8. Run `python utilities/stage_sites.py`.
+This script checks out `gh-pages` branch of the repository, builds the documentation site and javadocs, copies them to the branch and commits it. This script does not push the docs and it must be done manually on the later step. The script assumes that there is no directory called `tmp_gh-pages` in the repository root, remove that directory, if exists, before running the script.
 
-If you experience failures, you may need to:
-- repeat the clean step above
-- remove the temporary directory created to store docs by running `rm -rf tmp_gh-pages`
-- remove staged repositories from Sonatype (to prevent them from being released in subsequent steps): if a staged repository appears [here](https://oss.sonatype.org/#nexus-search;quick~com.google.cloud), remove it by running `mvn nexus-staging:drop`.
+9. Check that you are not trying to release a SNAPSHOT build (the artifacts versions do not have "-SNAPSHOT" suffix) and then run `mvn clean deploy -DskipTests=true --settings ~/.m2/settings.xml -P release` command. It will build and deploy artifacts to the staging repository. 
```

Comment:
It's mildly annoying that we have more steps now...

Reply:
I could and still can put all those in a single script. It is as easy as copy pasting steps commands in one script. Having more steps here can be considered as having more documentaiton on what is happening. The idea is to have all the "write" commands executed explicitly, so when somebody does release he knows for sure when things "get real".